pkg/app/master/commands/cliflags.go

```diff
@@ -37,6 +38,7 @@ const (
 	FlagLogLevelUsage      = "set the logging level ('trace', 'debug', 'info', 'warn' (default), 'error', 'fatal', 'panic')"
 	FlagLogUsage           = "log file to store logs"
 	FlagLogFormatUsage     = "set the format used by logs ('text' (default), or 'json')"
+	FlagConsoleOutputUsage = "set the format used by logs ('text' (default), or 'json')"
```

Comment:
should say: `set the console output format to use ('text' (default), or 'json')`

Reply:
Fixed it. Thanks!